doc/api/assert.md

```diff
@@ -310,6 +310,45 @@ parameter is undefined, a default error message is assigned. If the `message`
 parameter is an instance of an [`Error`][] then it will be thrown instead of the
 `AssertionError`.
 
+## assert.doesNotReject(block[, error][, message])
+<!-- YAML
+added: REPLACEME
+-->
+* `block` {Function}
+* `error` {RegExp|Function}
+* `message` {any}
+
+> Stability: 1 - Experimental
+
+Awaits for the promise returned by function `block` to complete and not be
+rejected. See [`assert.rejects()`][] for more details.
+
+When `assert.doesNotReject()` is called, it will immediately call the `block`
+function, and awaits for completion.
+
+Besides the async nature to await the completion it behaves identical to
+`assert.doesNotThrow()`.
```

Comment:
It would be nice to reference that function. And maybe also to add smth. like: `For further usage information, read that part.`

(The same applies to `assert.throws`).